store/wallets.go

```diff
@@ -0,0 +1,161 @@
+package store
+
+import (
+	"database/sql"
+	"errors"
+	"fmt"
+	"time"
+
+	"github.com/fewsats/fewsatscli/wallets"
+)
+
+// GetDefaultWallet returns the default wallet in the database.
+func (s *Store) GetDefaultWallet() (uint64, error) {
+	stmt := `
+		SELECT wallet_id
+		FROM default_wallet;
+	`
+
+	var id uint64
+	err := s.db.Get(&id, stmt)
+	if err != nil {
+		if errors.Is(err, sql.ErrNoRows) {
+			return 0, wallets.ErrNoWalletFound
+		}
+
+		return 0, err
+	}
+
+	return id, nil
+}
+
+// SetDefaultWallet sets the default wallet for the user.
+func (s *Store) SetDefaultWallet(walletID uint64) error {
+	stmt := `
+		DELETE FROM default_wallet;
+	`
+
+	_, err := s.db.Exec(stmt)
+	if err != nil {
+		return fmt.Errorf("failed to delete default wallet: %w", err)
+	}
+
+	stmt = `
+		INSERT INTO default_wallet (wallet_id)
+		VALUES ($1);
+	`
+
+	_, err = s.db.Exec(stmt, walletID)
+	if err != nil {
+		return fmt.Errorf("failed to set default wallet: %w", err)
+	}
+
```

Comment:
any reason not to do an update instead of delete + insert?

Reply:
If we do update, we need to be sure that it exits.  I was doing an upsert and was giving me shit so I went with delete + insert